src/main/java/core/basesyntax/FigureName.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum FigureName {
+    CIRCLE,
+    SQUARE,
+    RECTANGLE,
+    RIGHTTRIANGLE,
+    ISOSCELESTRAPESOID;

```

Comment:
According to the name convention, if constant contains more than one work, they should be separated by underscore - `_`
[See here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
    RIGHT_TRIANGLE,
    ISOSCELES_TRAPESOID;

```